src/api/public/apidocs-new/paths/distributions.yaml

```diff
@@ -0,0 +1,117 @@
+    Check /distributions/include_remotes for this.
+  security:
+    - basic_authentication: []
+  responses:
+    '200':
+      description: |
+        OK. The request has succeeded.
+
+        XML Schema used for body validation: [distributions.rng](../schema/distributions.rng)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/distributions.yaml'
+          example:
+            distribution:
+              - vendor: 'opensuse'
+                version: 'Tumbleweed'
+                id: 16626
+                name: 'openSUSE Tumbleweed'
+                project: 'openSUSE:Factory'
+                reponame: 'openSUSE_Tumbleweed'
+                repository: 'snapshot'
+                link: 'http://www.opensuse.org/'
+                icon:
+                  - url: 'https://static.opensuse.org/distributions/logos/opensuse.png'
+                    width: 8
+                    height: 8
+                  - url: 'https://static.opensuse.org/distributions/logos/opensuse.png'
+                    width: 16
+                    height: 16
+                architecture:
+                  - 'i586'
+                  - 'x86_64'
+              - vendor: 'openSUSE'
+                version: '15.3'
+                id: 16629
+                name: 'openSUSE Leap 15.3'
+                project: 'openSUSE:Leap:15.3'
+                reponame: 'openSUSE_Leap_15.3'
+                repository: 'standard'
+                link: 'http://www.opensuse.org/'
+                icon:
+                  - url: 'https://static.opensuse.org/distributions/logos/opensuse.png'
+                    width: 8
+                    height: 8
+                  - url: 'https://static.opensuse.org/distributions/logos/opensuse.png'
+                    width: 16
+                    height: 16
+                architecture: 'x86_64'
+              - vendor: 'Fedora'
+                version: '33'
+                id: 16731
+                name: 'Fedora 33'
+                project: 'Fedora:33'
+                reponame: 'Fedora_33'
+                repository: 'standard'
+                link: 'http://www.fedoraproject.org/'
+                icon:
+                  - url: 'https://static.opensuse.org/distributions/logos/fedora.png'
+                    width: 8
+                    height: 8
+                  - url: 'https://static.opensuse.org/distributions/logos/fedora.png'
+                    width: 16
+                    height: 16
+                architecture:
+                  - 'i586'
+                  - 'x86_64'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+  tags:
+    - Distributions
+
+put:
+  summary: Write all base distributions hosted on this OBS instace.
+  description: |
+    This Endpoint can be used to overwrite the list of all distributions hosted on this OBS instance.
+    Note this overwrites everything, it can't be used to add an additional distribution
```

Comment:
I don't understand. You want to say that this is for updating the attributes of all distributions but you can't add a new one?

Reply:
Well i guess you can of course add a new one but in that process you overwrite all existing ones. The point I wanted to make was that even if the xml contains the existing distributions it still overwrites them and e.g. gives them a new id.

Reply:
I guess just writing that it overwrites everything would be better because, yes in theory you could of course add an additional distribution.